cli/ballerina-cli-utils/src/main/java/org/ballerinalang/cli/utils/ExecutorUtils.java

```diff
@@ -42,13 +42,19 @@
      * Run balx that lives within jars.
      *
      * @param balxResource URI of the balx resource
+     * @param type         if a function or service is to be invoked
      * @param args         arguments passed to the function
      */
-    public static void execute(URI balxResource, String... args) {
+    public static void execute(URI balxResource, String type, String... args) {
         initFileSystem(balxResource);
         Path baloFilePath = Paths.get(balxResource);
         ProgramFile programFile = readExecutableProgram(baloFilePath);
-        BLangProgramRunner.runMain(programFile, args);
+
+        if (type.equals("function")) {
```

Comment:
not a good idea to use strings as types. let's use boolean or enum

Reply:
Fixed with 995c077